.github/workflows/release.yml

```diff
@@ -75,23 +100,20 @@ jobs:
               }
             }
             console.log(checksums)
-
-            return `${checksums['kapp-darwin-amd64']}  ./kapp-darwin-amd64
-            ${checksums['kapp-darwin-arm64']}  ./kapp-darwin-arm64
-            ${checksums['kapp-linux-amd64']}  ./kapp-linux-amd64
-            ${checksums['kapp-linux-arm64']}  ./kapp-linux-arm64
-            ${checksums['kapp-windows-amd64.exe']}  ./kapp-windows-amd64.exe`
+            return `${checksums['kapp-darwin-amd64']}  kapp-darwin-amd64
+            ${checksums['kapp-darwin-arm64']}  kapp-darwin-arm64
+            ${checksums['kapp-linux-amd64']}  kapp-linux-amd64
+            ${checksums['kapp-linux-arm64']}  kapp-linux-arm64
+            ${checksums['kapp-windows-amd64.exe']}  kapp-windows-amd64.exe`
 
       - name: verify uploaded artifacts
+        if: startsWith(github.ref, 'refs/tags/')
```

Comment:
I understand that we are trying to be extra cautious here but I think it's ok to have it. As manually also, we are building on two different machines and matching the checksum.

Reply:
+1 to what @joaopapereira has mentioned, I don't see any use of having this check.

> As manually also, we are building on two different machines and matching the checksum.

I think we initially did that as we saw the checksums mismatching from goreleaser compilation and local compilation, but I don't see how this would happen while directly using the binaries built from the scripts.

Reply:
Not the `if` loop, I think he is mentioning about `verifying the uploaded artifact` step.

Reply:
I was talking about the same `verifying the uploaded artifact`, but I am okay for being extra cautious.